packages/console/src/pages/Welcome/index.module.scss

```diff
@@ -6,32 +6,28 @@
   height: 100vh;
   flex-direction: column;
   min-width: 1280px;
+  min-height: 806px;
 
   &.light {
     background-image: url('../../assets/icons/welcome.svg');
-    background-position: right 0 bottom 40px;
+    background-position: right bottom;
+    background-size: contain;
     background-repeat: no-repeat;
```

Comment:
nit: can use a shared wrapper for these three props?

Reply:
updated